app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.framgia.soundclound.util;
+
+import android.databinding.BindingAdapter;
+import android.graphics.PorterDuff;
```

Comment:
ERROR: Unused import: android.graphics.PorterDuff. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports